charts/activemq/Chart.yaml

```diff
@@ -7,10 +7,10 @@ keywords:
 name: activemq
 sources:
   - https://github.com/Alfresco/alfresco-helm-charts
-version: 3.5.3
+version: 3.6.0
```

Comment:
Maybe we can bump to 3.5.4 since there is only a patch bump in dependencies

Reply:
Guys that does not comply to the defined versioning scheme: https://hyland.atlassian.net/wiki/spaces/TECH/pages/1293317631/Charts+release+versioning+scheme
I'm fine with changing it but I think it's time to review and approve (👍) that doc (it's still in "ready for review")

Reply:
looks more an uncovered case rather than not complying, what is the specific point you are referring to?

also we need to have this moved or at least mentioned in the developers/release md

Reply:
the page is indeed unclear but I'm prety sure we decided we would decoralate bugfixes from images and dependencies from actual chart bugfixes... Though thinking about it now seems overly complicated and treating all bugfixes the same is much easier. Hence, I updated the page and linked in the Release section.